CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(todocli LANGUAGES CXX)
+
```

Comment:
you can also set the c++ minimal version here: `set(CMAKE_CXX_STANDARD 17)`

(note: I'm JohnSquirrel from reddit, and I'm too lazy to make a PR now :))